permissions_validators/src/private_blockchain/query.rs

```diff
@@ -261,8 +280,43 @@ impl<W: WorldTrait> IsAllowed<W, QueryBox> for OnlyAccountsData {
             | FindAllAssets(_) => {
                 Err("Only access to the assets of the same domain is permitted.".to_owned())
             }
-            FindAllRoles(_) => Ok(()),
-            FindAllPeers(_) => Ok(()),
+            FindAllRoles(_)
+            | FindAllRoleIds(_)
+            | FindRoleByRoleId(_)
+            | FindAllPeers(_)
+            | FindAllActiveTriggerIds(_) => Ok(()),
+            FindTriggerById(query) => {
```

Comment:
I think all these are world-level access and should return `Err` as `OnlyAccountsData` means a restriction to account-level access